S3 Direct Upload.md

```diff
@@ -0,0 +1,9 @@
+# General
+
+S3 direct upload is the process of uploading files to the Amazon's S3 storage service.
+It is highly useful if you want to speed up the uploading process and to improve user experience.
+
+# The Workflow
+
+[This documentation](https://www.ironin.it/blog/store-your-files-on-s3-using-the-ruby-shrine-gem-part-2.html)
+covers all the code and configuration files you need for efficient direct upload to S3.
```

Comment:
promijenio bi
`...you need for efficient direct upload to S3. ...`
u
`needed for supporting direct upload to S3`

Reply:
Da, bolje zvuci!

Thanks! 👍